packages/xrpl/src/client/index.ts

```diff
@@ -639,14 +639,16 @@ class Client extends EventEmitter<EventTypes> {
    * Only used for multisigned transactions.
    * @returns The autofilled transaction.
    */
+  /* eslint-disable max-len -- Long linter directives are needed to perform type-erasure in this function */
+  // eslint-disable-next-line max-lines-per-function, complexity -- handling v2 Payment transaction API requires more logic
   public async autofill<T extends SubmittableTransaction>(
     transaction: T,
     signersCount?: number,
   ): Promise<T> {
     const tx = { ...transaction }
 
+    // the below two functions accept only Transaction objects.
```

Comment:
This comment is unnecessary.

Reply:
fixed in [5f218b2](https://github.com/XRPLF/xrpl.js/pull/2689/commits/5f218b2086887c0ae4746391eebfdca2e30b5909)